index.html

```diff
@@ -0,0 +1,215 @@
+      
+      <div><img src="./Images/snap02.png" alt="body image"></div>
+
+    <h2 class="multi-post">Multi-Post stories</h2>
+    <div class="list-1"> 
+      <div>
+      <ul class="list-1">
+        <li class="canopy">CANOPY</li>
+        <li class="dot"><img src="./Images/Counter.png" alt="Dot-1 image"></li>
+        <li class="job-title">Back End Dev</li>
+        <li><img src="./Images/Counter.png" alt="Dot-2 image"></li>
+        <li class="year">2015</li>
+      </ul>
+      </div>
+    </div>
+
+    <p class="short-info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+    <ul class="buttons">
+      <li><button class="html-btn">html</button></li>
+      <li><button class="css-btn">css</button></li>
+      <li><button class="js-btn">javascript</button></li>
+    </ul>
+    <div class="project-btn"><button class="pro-btn">See Project</button></div>
+    </div>
+  </section>
+
+
+
+  <section>
+    <div class="main-body1">
+      
+      <div><img src="./Images/snap1.png" alt="body image"></div>
+
+    <h2 class="tonic">Tonic</h2>
+    <div class="list-1"> 
+      <ul class="list-1">
+        <li class="canopy">CANOPY</li>
+        <li class="dot"><img src="./Images/Counter.png" alt="Dot-1 image"></li>
+        <li class="job-title">Back End Dev</li>
+        <li><img src="./Images/Counter.png" alt="Dot-2 image"></li>
+        <li class="year">2015</li>
+      </ul>
+    </div>
+
+    <p class="short-info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+    <ul class="buttons">
+      <li><button class="html-btn">html</button></li>
+      <li><button class="css-btn">css</button></li>
+      <li><button class="js-btn">javascript</button></li>
+    </ul>
+    <div class="project-btn"><button class="pro-btn">See Project</button></div>
+    </div>
+  </section>
+
+  <section>
+    <div class="main-body1">
+      
+      <div><img src="./Images/snap2.png" alt="body image"></div>
+
+    <h2 class="multi-post">Multi-Post stories</h2>
+    <div class="list-1"> 
+      <div>
+      <ul class="list-1">
+        <li class="canopy">CANOPY</li>
+        <li class="dot"><img src="./Images/Counter.png" alt="Dot-1 image"></li>
+        <li class="job-title">Back End Dev</li>
+        <li><img src="./Images/Counter.png" alt="Dot-2 image"></li>
+        <li class="year">2015</li>
+      </ul>
+      </div>
+    </div>
+
+    <p class="short-info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+    <ul class="buttons">
+      <li><button class="html-btn">html</button></li>
+      <li><button class="css-btn">css</button></li>
+      <li><button class="js-btn">javascript</button></li>
+    </ul>
+    <div class="project-btn"><button class="pro-btn">See Project</button></div>
+    </div>
+  </section>
+  
+  </main>
+
+  <section>
+    <div class="grid-container2">
+      <h1 class="about-heading">About Myself</h1>
```

Comment:
- [ ] Please, kindly use another header tag, as your page can only have one h1 tag. fix this to improve search engine optimization on your website

Reply:
- [ ] As suggested by the previous code reviewer, there should be only one `h1` in an HTML file because of semantics. I kindly suggest that you make only your main title of "I'm Zeinab Glad to see you!" an `h1` and kindly make use of headers from `h2` down to `h6` for any other headers in your HTML file.


### EXAMPLE
`<h2 class="about-heading">About Myself</h2>
`